### ISSUE TITLE
Can't load MayaUSD plugin in Maya 2022:

### ISSUE THREAD

**Issue by scratchapixel** [Autodesk/maya-usd#2199]
**Describe the issue**
I seem to have a similar issue to this one:

https://github.com/Autodesk/maya-usd/issues/1389

Using Maya 2022 and trying to load the pre-built plugin MayaUSD. I used the wizard to install them here:

C:\Program Files\Autodesk\MayaUSD\Maya2020\0.16.0_202202052333-1c49ce6\

I set the MAYA_MODULE_PATH to C:\Program Files\Autodesk\MayaUSD\Maya2020\0.16.0_202202052333-1c49ce6. This work, as I see the plugin in the plugin manager (see screenshot below). If I try load the plugin manually from Maya I get the following error:

    // Error: file: D:/Program Files/Autodesk/Maya2022/scripts/others/pluginWin.mel line 934: The specified module could not be found.
     (mayaUsdPlugin) // 
    // Warning: file: D:/Program Files/Autodesk/Maya2022/scripts/others/pluginWin.mel line 937: Could not load C:/Program 
    Files/Autodesk/MayaUSD/Maya2020/0.16.0_202202052333-1c49ce6/mayausd/MayaUSD/plugin/adsk/plugin/mayaUsdPlugin.mll as a plug-in // 

Auto-load doesn't work either.

I've built USD on this machine before but deleted all variables that could interfere with it (PATH, PYTHON_PATH).

I am not sure to see what would have been the solution to the referenced issue as I don't have Nuke in the path. Any help would be greatly appreciated).

**Specs:**
- Windows 10
- Maya 2022
- MayaUSD 0.16.0_202202052333

**Additional context**
Content of $PATH

/c/Users/corgi/bin:/mingw64/bin:/usr/local/bin:/usr/bin:/bin:/mingw64/bin:/usr/bin:/c/Users/corgi/bin:/d/VulkanSDK/1.2.182.0/Bin:/d/VulkanSDK/1.2.176.1/Bin:/c/Program Files (x86)/Common Files/Intel/Shared Libraries/redist/intel64_win/compiler:/c/Program Files (x86)/Common Files/Intel/Shared Libraries/redist/intel64/compiler:/c/WINDOWS/system32:/c/WINDOWS:/c/WINDOWS/System32/Wbem:/c/WINDOWS/System32/WindowsPowerShell/v1.0:/c/Program Files (x86)/Microsoft Emulator Manager/1.0:/cmd:/c/Program Files (x86)/GtkSharp/2.12/bin:/c/Program Files/dotnet:/c/Program Files/Common Files/Autodesk Shared:/c/Program Files (x86)/Autodesk/Backburner:/c/Program Files (x86)/NVIDIA Corporation/PhysX/Common:/c/WINDOWS/System32/OpenSSH:/c/Program Files (x86)/GitExtensions:/c/TDM-GCC-64/bin:/c/Program Files (x86)/Google/Cloud SDK/google-cloud-sdk/bin:/c/Program Files/PuTTY:/c/Program Files/tools:/c/Program Files/Microsoft SQL Server/120/Tools/Binn:/c/Program Files/NVIDIA Corporation/NVIDIA NvDLISR:/c/Program Files/Microsoft VS Code/bin:/c/WINDOWS/system32:/c/WINDOWS:/c/WINDOWS/System32/Wbem:/c/WINDOWS/System32/WindowsPowerShell/v1.0:/c/WINDOWS/System32/OpenSSH:/c/Program Files/CMake/bin:/c/Users/corgi/Documents/maya:/d/Program Files/LLVM/bin:/c/Users/corgi/AppData/Local/Microsoft/WindowsApps:/c/Program Files (x86)/Nmap:/c/Program Files/Microsoft VS Code/bin:/c/bin:/d/flutter/bin:/c/Users/corgi/.dotnet/tools:/usr/bin/vendor_perl:/usr/bin/core_perl

![image](https://user-images.githubusercontent.com/5096025/158996141-70ecd02c-aff6-49ef-b055-165550145aa7.png)

**Comment by ppt-adsk** [comment]
Hi @scratchapixel, the environment variable for the Python path is actually PYTHONPATH, not PYTHON_PATH.  Make sure you have that all checked out.  Here's another similar discussion:
https://github.com/Autodesk/maya-usd/issues/1615

**Comment by seando-adsk** [comment]
One thing that can help us find the problem is if you can post the output of the commands: `getenv "PATH"` and `getenv "PYTHONPATH"` from the Script Editor in Maya.

Sean

**Comment by scratchapixel** [comment]
Sean. For now I have installed Maya 2022.3 which comes with MayaUSD so that I could get started right away. Sorry just didn't have the patience to work this problem out)(. I am about to reconfigure the USD build after I have installed Maya 2022.3 so hope this won't cause any additional friction. Thanks again for your help (yes I meant PYTHONPATH and not PYTHON_PATH). But I will give it another go later...
